exporter/opentelemetry-exporter-datadog/tests/test_datadog_format.py

```diff
@@ -22,9 +22,18 @@
 FORMAT = propagator.DatadogFormat()
 
 
-def get_as_list(dict_object, key):
-    value = dict_object.get(key)
-    return [value] if value is not None else []
+class Getter:
```

Comment:
Shouldn't we be testing the actual implementation of the `Getter` used instead of a differently defined one in the test?